pkg/crypto/crypto.go

```diff
@@ -26,8 +36,14 @@ func OpenVolume(volume, devicePath, passphrase string) error {
 		return nil
 	}
 
+	namespaces := []lhtypes.Namespace{lhtypes.NamespaceMnt, lhtypes.NamespaceIpc}
+	nsexec, err := lhns.NewNamespaceExecutor(lhtypes.ProcessNone, lhtypes.HostProcDirectory, namespaces)
+	if err != nil {
+		return err
+	}
+
 	logrus.Debugf("Opening device %s with LUKS on %s", devicePath, volume)
-	_, err := luksOpen(volume, devicePath, passphrase)
+	_, err = nsexec.LuksOpen(volume, devicePath, passphrase, lhtypes.LuksTimeout)
```

Comment:
We don't log the stdout and, especially, stderr of LuksOpen(). Can you help create a ticket for improvement? Thank you.

Reply:
Sure, created
https://github.com/longhorn/longhorn/issues/8171